### PR TITLE
Lighthouse fixes

### PR DESCRIPTION
**Pull request by juliantoledo**
We are storing more information from the Lighthouse report to be used in the details page.

### CODE REVIEW

File: lib/lighthouse.js [hunk 1]
```diff
@@ -81,28 +110,42 @@ exports.fetchAndSave = function(pwaId) {
  */
 exports.processLighthouseJson = function(lighthouseJson) {
   var lighthouseInfo = {};
-  lighthouseJson.aggregations.forEach(section => {
+  lighthouseJson.aggregations.forEach(agregation => {
```
Comment (andreban) [inline lib/lighthouse.js]: s/agregation/aggregation

File: lib/lighthouse.js [hunk 2]
```diff
@@ -81,28 +110,42 @@ exports.fetchAndSave = function(pwaId) {
  */
 exports.processLighthouseJson = function(lighthouseJson) {
```
Comment (andreban) [inline lib/lighthouse.js]: This method has many instances of the usage of 'var'. We should always use 'const' / 'let'

File: lib/lighthouse.js (hunk 1 above)
Comment (juliantoledo) [inline lib/lighthouse.js]: done.

File: lib/lighthouse.js (hunk 2 above)
Comment (juliantoledo) [inline lib/lighthouse.js]: @andreban I will leave this change for #166.

**Comment by andreban** [comment]
LGTM

**Pull request merged**